### PR TITLE
add attributes

### PR DESCRIPTION
**Pull request by kevinch**
Add attribute for e2e testing.

### CODE REVIEW

File: react/components/Iframe.js [hunk 1]
```diff
@@ -166,6 +202,7 @@ class Iframe extends Component {
         ref={this.handleRef}
         onLoad={this.handleOnLoad}
         data-hj-suppress
+        data-testid="admin-frame-container"
```
Comment (matheusps) [inline react/components/Iframe.js]:
```suggestion
        data-testid="admin-iframe-container"
```

File: react/components/IframeLegacy.js [hunk 2]
```diff
@@ -168,6 +168,7 @@ class IframeLegacy extends Component {
           ref={this.handleRef}
           onLoad={this.handleOnLoad}
           data-hj-suppress
+          data-testid="admin-frame-container-legacy"
```
Comment (matheusps) [inline react/components/IframeLegacy.js]:
```suggestion
          data-testid="admin-iframe-container-legacy"
```

**Review by matheusps — COMMENTED** [review]
Maybe?

File: react/components/IframeLegacy.js (hunk 2 above)
Comment (kevinch) [reply react/components/IframeLegacy.js]: Good catch! Don't know what happened here.

File: react/components/Iframe.js (hunk 1 above)
Comment (kevinch) [reply react/components/Iframe.js]: Good catch! Don't know what happened here.

**Review by matheusps — APPROVED** [review]

**Review by marcelovicentegc — APPROVED** [review]

**Pull request merged**